power_grid_model_c/power_grid_model/include/power_grid_model/three_phase_tensor.hpp

```diff
@@ -125,17 +123,11 @@ static_assert(std::is_trivially_destructible_v<ComplexValue<false>>);
 
 // enabler
 template <class T>
-constexpr bool check_vector_v = T::ColsAtCompileTime == 1;
-template <class T>
-constexpr bool check_tensor_v = (Idx)T::RowsAtCompileTime == (Idx)T::ColsAtCompileTime;
-template <class T>
-constexpr bool check_all_v = check_tensor_v<T> || check_vector_v<T>;
-template <class T>
-using enable_vector_t = std::enable_if_t<check_vector_v<T>>;
+concept column_vector = (T::ColsAtCompileTime == 1);
 template <class T>
-using enable_tensor_t = std::enable_if_t<check_tensor_v<T>>;
+concept rk2_tensor = (static_cast<Idx>(T::RowsAtCompileTime) == static_cast<Idx>(T::ColsAtCompileTime));
```

Comment:
What does `rk2_tensor` mean? Maybe add a comment line? `rk2` does not seem to be a common abbriviation.

Reply:
rank 2 tensor. i did not want to make it bare `tensor` because that could easily conflict with other names and we explicitly demand that it has 2 dimensions (rows and cols). happy to add a comment or use a different name (or both). your call

Reply:
Just add a comment. The name is fine.